searx/engines/freesound.py

```diff
@@ -0,0 +1,67 @@
+"""
+Freesound (Sound)
+"""
+
+from json import loads
+from urllib.parse import urlencode
+from datetime import datetime
+
+disabled = True
+api_key = ""
+
+# about
+about = {
+    "website": "https://freesound.org",
```

Comment:
can add the field `"wikidata_id": "Q835703",`

(see https://www.wikidata.org/wiki/Q835703 )

Reply:
done